### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by Minghui-Gao**
users can see and edit their own profile.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -43,5 +49,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```
Comment (LiamANeeson) [inline client/package.json:53]: We are already using bootstrap avoid using tailwind. Better to only use one way for styling

**Review by LiamANeeson — COMMENTED** [review]
If we can avoid conflicting using multiple frameworks for styling that would be better. Bootstrap has already been used so you can redesign with Bootstrap that would be optimal, thanks!

File: client/src/Pages/Profile/Profile.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
```
[the remark this answers is not in the document]
Comment (Minghui-Gao) [reply client/src/Pages/Profile/Profile.css:1]: Okay. Thanks! I will uninstall the tailwind and change to use Bootstrap

**Comment by LiamANeeson** [comment]
Something like this may be useful for creating a profile page via bootsrap: 

1. https://mdbootstrap.com/docs/standard/extended/profiles/
2. https://freefrontend.com/bootstrap-profiles/
3. https://www.youtube.com/watch?v=bamVA74eh0k

**Comment by Minghui-Gao** [comment]
> Something like this may be useful for creating a profile page via bootstrap:
> 
> 1. https://mdbootstrap.com/docs/standard/extended/profiles/
> 2. https://freefrontend.com/bootstrap-profiles/
> 3. https://www.youtube.com/watch?v=bamVA74eh0k

Thank you very much, helpful to me.

**Review by LiamANeeson — APPROVED** [review]
LGTM

**Comment by Minghui-Gao** [comment]
> 

users can change their avatar by uploading the file but thay cannot see the uploaded file

**Pull request merged**